### DISCUSSION TITLE
PIXEL_COLOUR_DEPTH_BITS naming changes

### DISCUSSION THREAD

**Discussion by hpsauer** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#380] [Q&A] [answered]
Hello,

I started to review an old project using the ESP32-HUB75 library, Arduino IDE told that there is a library update 3.0.5 available for it. 
Done as reuested but on the next compile it failed telling that PIXEL_COLOR_DEPTH_BITS is not declared.
After checking wasting time, I found that there has a naming change made to PIXEL_COLOUR_DEPTH_BITS. 
Why suddenly does someone changed that without any notice in a README about that changes regarding English/US spelling differences? Any other one also being faced on that?
In any other text and descriptions, the wording used is still COLOR...

Greetings
HP Sauer

**Comment by mrcodetastic** [comment, marked as answer]
I'll take your constructive criticism on board and be more careful to update the documentation when I make changes to the library in my spare time. Sorry for the time wasted 

In saying that, I'll probably revert to what it was ('color') for consistency with the color functions.
